tools/doc/tests/test_html.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+
+var assert = require('assert');
+var fs = require('fs');
+
+var html = require('../html.js');
+
+// Outputs valid html when given simple markdown
+function test_toHTML() {
+  // Test data is a list of objects with two properties.
+  // The file property is the file path.
+  // The html property is some html which will be generated by the doctool.
+  // This html will be stripped of all whitespace.
```

Comment:
Maybe change the comment to state that it will be stripped of all unnecessary whitespace?